.github/workflows/test.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           cloud-url: file://~
           stack-name: dev
           upsert: true
+          downsert: true
```

Comment:
I don't think we need to include downserting in our own workflows. I don't think we want to lose the stack history.

Reply:
Done. I've reverted [the workflow commit](https://github.com/pulumi/actions/commit/487ad9f6f0b157d1933172f6812c07e16eefa5e0).